colourblind8/colourblind8.py

```diff
@@ -1,8 +1,51 @@
 import matplotlib.pyplot as plt
 
 class Colourblind8:
+	""" What is Colourblind8?
+    
+    Colourblind8 provides an improved version of matplotlib scatterplot,lineplot and histogram through colour and design enhancement 
+    for three types of colourblind conditions - Deuteranopia,Protanopia,Tritanopia.
+
+    Example:
+    
+        Colourblind8.plot_lines(x=x, y=y, alpha=1.0,labels =['a','b','c','d','e','f','g','h','i'], palette = 'deutera', 
+                                title = "My testing", x_lab = "X_test", y_lab = "Y_test", legend_title = "lengendtest");
+    Usage:
+    
+        - plot selection: 
+        
+            Colourblind8.plot_lines()
+            Colourblind8.plot_scatter()
+            Colourblind8.plot_hist()
+    
+        - Colourpalette selection through `palette` argument within `plot()`:
+    
+            palette=['deutera_colours','prota_colours','trita_colours'] 
+    
+    Args:
+        param1 x (int): a list of numeric values. Applicable to scatterplots and lineplots
+        param2 y (list of lists): up to 9 y variables to plot based on the number of available colours per palette. 
+			For one list, please convert it to a list of lists.  
+        alpha(default value =1.0): transparency level, it takes any value between 0.0 and 1.0.
+        labels : Optional. Labels for each object within the graph.
+        palette (one palette): 'deutera_colours', 'prota_colours','trita_colours'
+        title: Optional. Title for the plot
+        x_lab: Optional. Label for x axis
+        y_lab: Optional. Label for y axis
+        legend_title: Optional. Legend title for legend.
 
+    Returns:
+	
+		an enhanced version of matplotlib plot.
```

Comment:
Add the return type of the matplotlib. Also make sure that the documentation follows the basic Numpy Styling. Available here: https://www.numpy.org/devdocs/docs/howto_document.html#docstring-standard

Reply:
Thank you for the suggestion! I will reflect this in my next pull request.